cloudmarker/events/vmdatadiskencryptionevent.py

```diff
@@ -0,0 +1,122 @@
+"""VM Data disk encryption event.
+
+This module defines the :class:`VMDataDiskEncryptionEvent` class that
+identifies an unencrypted VM data disk. This plugin works on the virtual
+machine properties found in the ``com`` bucket of ``virtual_machine``
+records.
+"""
+
+
+import logging
+
+from cloudmarker import util
+
+_log = logging.getLogger(__name__)
+
+
+class VMDataDiskEncryptionEvent:
+    """VM Data disk encryption event plugin."""
+
+    def __init__(self):
+        """Create an instance of :class:`VMDataDiskEncryptionEvent`."""
+
+    # pylint: disable=R0201
+    def eval(self, record):
+        """Evaluate virtual machine to check for unencrypted data disks.
+
+        Arguments:
+            record (dict): A virtual machine record.
+
+        Yields:
+            dict: An event record representing an unencrypted data disk
+            of a virtual machine
+
+        """
+        # If 'com' bucket is missing, we have a malformed record. Log a
+        # warning and ignore it.
+        com = record.get('com')
+        if com is None:
+            _log.warning('Virtual machine record is missing \'com\' key: %r',
+                         record)
+            return
+
+        # This plugin understands compute rule records only,
+        # so ignore  any other record types.
+        common_record_type = com.get('record_type')
+
+        if common_record_type != 'compute':
+            return
+
```

Comment:
I think we can put `if com.get('cloud_type') != 'azure': return` here itself. The code at line 61 below would become simpler then with one indentation level less.

Reply:
Having  `if com.get('cloud_type') != 'azure': return` results in the following issue:
`E701 multiple statements on one line (colon) [pycodestyle]`
Suggestion?